assets/js/base/context/cart-checkout/event-emit/emitters.js

```diff
@@ -1,7 +1,7 @@
 const getObserversByPriority = ( observers, eventType ) => {
 	return observers[ eventType ]
 		? Array.from( observers[ eventType ].values() ).sort( ( a, b ) => {
-				return b.priority - a.priority;
+				return a.priority - b.priority;
```

Comment:
This order was wrong (and I noticed that the test expectation was wrong!). The intent is that _lower_ priority observers get emitted before higher priority.